### ISSUE TITLE
[Bug]: ccache: error: failed to link

### ISSUE THREAD

**Issue by rendiix** [termux/termux-packages#7454]
### Problem description

ccahe failed to link.
I can't use ccache in CMake or autotols.

### What steps will reproduce the bug?

compile comand

```
$ ccache clang -c hello.c 
ccache: error: failed to link /data/data/com.termux/files/home/.cache/ccache/tmp/tmp.cpp_stdout.WcUq5t to /data/data/com.termux/files/home/.cache/ccache/tmp/tmp.cpp_stdout.WcUq5t.i: Permission denied
```

Ccache default config:

```
$ ccache -ps
(default) absolute_paths_in_stderr = false
(default) base_dir = 
(default) cache_dir = /data/data/com.termux/files/home/.cache/ccache
(default) compiler = 
(default) compiler_check = mtime
(default) compiler_type = auto
(default) compression = true
(default) compression_level = 0
(default) cpp_extension = 
(default) debug = false
(default) debug_dir = 
(default) depend_mode = false
(default) direct_mode = true
(default) disable = false
(default) extra_files_to_hash = 
(default) file_clone = false
(default) hard_link = false
(default) hash_dir = true
(default) ignore_headers_in_manifest = 
(default) ignore_options = 
(default) inode_cache = false
(default) keep_comments_cpp = false
(default) limit_multiple = 0.8
(default) log_file = 
(default) max_files = 0
(default) max_size = 5.0G
(default) path = 
(default) pch_external_checksum = false
(default) prefix_command = 
(default) prefix_command_cpp = 
(default) read_only = false
(default) read_only_direct = false
(default) recache = false
(default) run_second_cpp = true
(default) sloppiness = 
(default) stats = true
(default) temporary_dir = /data/data/com.termux/files/home/.cache/ccache/tmp
(default) umask = 
cache directory                     /data/data/com.termux/files/home/.cache/ccache
primary config                      /data/data/com.termux/files/home/.config/ccache/ccache.conf
secondary config (readonly)         /data/data/com.termux/files/usr/etc/ccache.conf
stats updated                       Thu Sep  2 18:59:33 2021
cache hit (direct)                     0
cache hit (preprocessed)               0
cache miss                             0
cache hit rate                      0.00 %
called for link                        2
bad compiler arguments                 1
autoconf compile/link                  3
no input file                          4
cleanups performed                     0
files in cache                         0
cache size                           0.0 kB
max cache size                       5.0 GB
```

### System information

termux-info:

``` 
Application version:
0.117
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages.termux.org/apt/termux-main/ stable main
# game-repo (sources.list.d/game.list)
deb https://packages.termux.org/apt/termux-games games stable
# science-repo (sources.list.d/science.list)
deb https://packages.termux.org/apt/termux-science science stable
# sources.list.d/rendiix.list
deb https://rendiix.github.io android-tools termux
Updatable packages:
libprotobuf/stable 2:3.17.3 aarch64 [upgradable from: 3.9.1]
Android version:
10
Kernel build information:
Linux localhost 4.14.117-perf #1 SMP PREEMPT Mon Jul 19 17:49:04 CST 2021 aarch64 Android
Device manufacturer:
asus
Device model:
ASUS_I001DE

```

**Comment by Grimler91** [comment]
Should be fixed by https://github.com/termux/termux-packages/commit/a3c7f9fb416dd29420cc65803f68298d7b1ca6ff, let me know if it still does not work

**Comment by rendiix** [comment]
> Should be fixed by [a3c7f9f](https://github.com/termux/termux-packages/commit/a3c7f9fb416dd29420cc65803f68298d7b1ca6ff), let me know if ccache still has issues

thanks for fixing
